components/src/dynamo/planner/config/planner_config.py

```
@@ -49,6 +49,22 @@ def _prometheus_ssl_verify_default() -> bool:
     )
 
 
+def _gcd_seconds(values: list[float]) -> float:
+    """Return a millisecond-resolution gcd for second-based intervals."""
+    millis = [int(round(v * 1000.0)) for v in values]
```

Ruff (0.15.15)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)